src/WasmExecutor.cpp

```diff
@@ -594,67 +578,67 @@ JITUserContext *get_jit_user_context(WabtContext &wabt_context, const wabt::inte
     if (ucon_magic == 0) {
         return nullptr;
     }
-    wassert(ucon_magic == kMagicJitUserContextValue);
+    internal_assert(ucon_magic == kMagicJitUserContextValue);
     JITUserContext *jit_user_context = wabt_context.jit_user_context;
-    wassert(jit_user_context);
+    internal_assert(jit_user_context);
     return jit_user_context;
 }
 
 void dump_hostbuf(WabtContext &wabt_context, const halide_buffer_t *buf, const std::string &label) {
-#if WASM_DEBUG_LEVEL >= 2
+#if DUMP_HOSTBUFS
```

Comment:
```suggestion
#if DUMP_HOST_BUFFERS
```

I think this is the intended symbol name?